config/cookstyle.yml

```diff
@@ -2,7 +2,7 @@ AllCops:
   SuggestExtensions: false
   NewCops: disable
   DisabledByDefault: true
-  TargetRubyVersion: 2.5
+  TargetRubyVersion: 2.6
```

Comment:
wait, why is it 2.6 here and 3.0 above?

Reply:
Cookstyle vs. Chefstyle

Reply:
Isn't that just the ruby version whose "cops" are applied to your code? You're apply Ruby 2.6 syntax-checking and linting. I would expect Ruby 3.0 as well

Reply:
Some cops will try to modernize your code with new methods that don't exist in old Ruby releases. If we set that to the latest and greatest Ruby releases you couldn't modernize the Chef side without introducing Ruby that old clients couldn't use. Folks can always crank that up as necessary. Ruby 2.6 is Infra Client 15.